src/app/organization/sms-campaigns/view-sms-campaign/campaign-tab/campaign-tab.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="30px">
+    <button mat-raised-button color="primary">
+    <i class="fa fa-remove"></i>
+    &nbsp;&nbsp; Close
```

Comment:
you need to add a mat dialog for this
![Screenshot from 2020-05-07 02-00-05](https://user-images.githubusercontent.com/54709463/81225247-8859a000-9006-11ea-9b65-d1940b97b356.png)

Reply:
@abhaychawla Do I need to implement the functionality of buttons in this PR or is that a separate issue?